### PR TITLE
fix missing stdlibs; update bin/*.toml files

### PR DESCRIPTION
**Pull request by StefanKarpinski**
`OldPkg` and `Sockets` were missing from the list of stdlibs in the `bin` scripts. The `bin/sha1map.toml` and `bin/stdlib.toml` maps also hadn't been updated in a long time so this took a really long time to run from scratch. @KristofferC, maybe it would be good to push updates from the version of this script that you run regularly? Also, what do I need to do to get this change to be picked up there?

### CODE REVIEW

File: bin/gitmeta.jl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import Pkg.TOML
 import LibGit2
 
-const STDLIBS = [
+const STDLIBS = sort!(by = lowercase, [
```
Comment (fredrikekre) [inline bin/gitmeta.jl]: TIL you can use kwargs like this, thats pretty neat.

Comment (StefanKarpinski) [reply bin/gitmeta.jl]: Yup, keyword and positional args can be interleaved, a nice aspect of keeping them distinct.

**Comment by KristofferC** [comment]
Please squash merge or rebase merge. Makes it easier to cherry-pick.

**Comment by StefanKarpinski** [comment]
This was squashed.

**Comment by KristofferC** [comment]
Sorry, it was the `compat` rename PR.

**Comment by StefanKarpinski** [comment]
Yeah, I'm not used to rebase being an option since it was turned off on JuliaLang/julia for so long.

**Pull request merged**